parts/dcoscustomdata187.t

```diff
@@ -337,3 +337,5 @@ write_files:
   path: "/opt/azure/containers/provision.sh"
   permissions: "0744"
   owner: "root"
+- path: /var/lib/dcos/mesos-slave-common

```

Comment:
are you also going to add this to dcoscustomdata173.t or validation to block 1.7.3's use of attributes?